crates/runtimes/src/lib.rs

```diff
@@ -33,7 +35,7 @@ pub fn init_runtime(
             other => init_external_runtime(project_root, config, path, other),
         }
     } else {
-        Err(anyhow!("The given file does not have a valid extension"))
+        Err(errors::RuntimeError::FileHasInvalidExtension { extension: None })
```

Comment:
Apply same changes on the names:

```suggestion
        Err(errors::RuntimeError::InvalidExtension { extension: None })
```